js/DOM.js

```diff
@@ -71,3 +71,7 @@ const domUpdates = {
 
 
 
+
+
+
+
```

Comment:
Try not to have random diffs like this in your pull requests, they clutter things up and make it a little harder to read.